Dev-Corner/slides/tool_integration.html

```diff
@@ -1520,10 +1627,108 @@
 
                     <section data-markdown>
                         <script type="text/template">
-                            ### `macro`
+                            ### `macros`
 
                             <img src="../images/macro.png" alt="macro"/>
 
+                            <small>[Planemo documents about macros ](http://planemo.readthedocs.io/en/latest/_writing_suites.html#macros)</small>
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### `macros`
+                            #### XML elements
+
+                            <div align="left">
+                                macros.xml
+                            </div>
+
+                            ```xml
+                            <macros>
+                              <xml name="requirements">
+                                <requirements>
+                                  <requirement type="package" version="2.5.0">blast</requirement>
+                                </requirements>
+                              </xml>
+                              <xml name="stdio">
```

Comment:
Because it's exactly the kind of tag you include in a macro.
I didn't have the place to add the `<expand macro="stdio"/>`